SetupIncTraceLibs.bat

```diff
@@ -0,0 +1,155 @@
+@echo off

```

Comment:
Open to suggestions on how we should expose this logic. Until we expose it to end users, I'm not sure we even want allow users access to the worker packages.

Reply:
It would be nice to reuse Setup.bat somehow, instead of duplicating the code. But that wouldn't block the PR.

Reply:
I've reworked this to reuse setup.bat now.